tutorials/single_node_tutorial/single_gpu_tutorial.ipynb

```diff
@@ -0,0 +1,3423 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "1c1a4119",
+   "metadata": {},
+   "source": [
+    "# Nemo Curator pipeline example\n",
+    "\n",
+    "## NeMo Curator introduction\n",
+    "The NeMo Curator is a Python library that consists of a collection of scalable data-mining modules for curating natural language processing (NLP) data for training large language models (LLMs). The modules within the NeMo Data Curator enable NLP researchers to mine high-quality text at scale from massive uncurated web corpora. \n",
+    "\n",
+    "NeMo Curator includes the following modules to perform data curation:\n",
+    "- Data download and Extraction\n",
+    "- Language identification and separation\n",
+    "- Text reformatting and cleaning\n",
+    "- Quality filtering\n",
+    "- Document-level deduplication\n",
+    "- Multilingual downstream-task decontamination\n",
+    "- Distributed Data Classification\n",
+    "\n",
+    "NeMo Curator team has perform ablation experiments using Common Crawl dataset to train a 357M GPT-style model to assess the effect of different curation stage on model performance. \n",
+    "\n",
+    "![alt text](./image/zeroshot_ablations.png)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "be41377f",
+   "metadata": {},
+   "source": [
+    "## About this notebook\n",
+    "\n",
+    "\n",
+    "This notebook will use **Thai Wikipedia dataset** as example to demonstrate a typical data curation pipeline using NeMo Curator. After running through this script, user will be able to know how to use NDC to download wikipedia data, perform language separation using fasttext, perform GPU based exact deduplication and fuzzy deduplication and use CPU based heuristic filtering. \n",
+    "\n",
+    "Step description:\n",
+    "1. Download and extract data\n",
+    "2. Language detection and separation\n",
+    "3. GPU based deduplication\n",
+    "    1. Exact deduplication\n",
+    "    2. Fuzzy deduplication\n",
+    "4. Heuristic filtering\n",
+    "\n",
+    "What is not included:\n",
```

Comment:
Probably also worth mentioning that this also doesn't include
1. Distributed data classification with PyTorch models
2. Personal identifiable information (PII) redaction

Reply:
Added